### DISCUSSION TITLE
Outdated webdigest

### DISCUSSION THREAD

**Discussion by macmiranda** [neruthes/webdigest#2] [General]
Hey,

It seems https://webdigest.pages.dev/ stopped being updated though the markdown files in the repo are up-to-date

It would be nice if they were in sync

Anything I can do to help?

**Comment by neruthes** [comment]
It is nice to know that you noticed this project. I occasionally encounter attention/depression problems which stop me from fixing problems in "old" projects. Your message may prove to be a great contribution in restoring motivation. Thank you.

I will write back if I cannot manage fixing the building workflow on my own. 🚀

**Comment by neruthes** [comment]
I looked into the logs and figured out that the deploying process was blocked by an error where the PDF collection package tarball exceeds the file size limit on Cloudflare R2. Luckily, the building process was good.

After some manual intervention, the website now exhibits the latest PDF as of 2023 Dec 25. I hope that the automatic build for 2023 Dec 26 can succeed. If things got right, we will see the PDF artifact uploaded and linked on the website by Dec 26 00:03 UTC.

**Comment by neruthes** [comment]
The 2023-12-26 artifact has been uploaded. The RSS parser script complained about invalid XML during the automatic build, but I cannot reproduce manually. I will see how it goes tomorrow.

**Comment by macmiranda** [comment]
It seems the rss build time (`Wed Dec 27 00:00:21 UTC 2023`) is before the actual publishing time (`Tue Dec 26 00:01:00 UTC 2023`) of the digest which delays the update by 24h
